autoload/wilder/renderer/nvim_api.vim

```diff
@@ -86,7 +86,7 @@ function! s:_open_win() dict abort
   endif
 
   " Fix E5555 when re-showing wilder when inccommand is cancelled.
-  let l:buf = has('nvim-0.6') ? 0 : self.state.buf
+  let l:buf = has('nvim-0.7') && !has('nvim-0.7') ? 0 : self.state.buf
```

Comment:
Is this line intended or a mistake? (L108 as well)

Reply:
Thanks for catching this! Fixed this in 9c33d9423a3ba205ecdb90ce8a677c2b26f04908.